test/NuGet.Clients.Tests/NuGet.CommandLine.Test/Util.cs

```diff
@@ -1274,5 +1274,174 @@ public static void CreateTempGlobalJson(string solutionRoot)
 
         }
 
+        #region helper methods for .NET nanoFramework tests
+
+        /// <summary>
+        /// Create a basic nfproj file for .NET nanoFramework.
+        /// </summary>
+        public static string GetNFProjXML(
+            string projectName,
+            List<(string, string)> packages)
+        {
+            StringBuilder projContent = new StringBuilder();
+
+            projContent.AppendLine(
+@"
+<?xml version=""1.0"" encoding=""utf-8""?>
+    <Project ToolsVersion=""Current"" DefaultTargets=""Build"" xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"">
+        <PropertyGroup Label=""Globals"">
+        <NanoFrameworkProjectSystemPath>$(MSBuildToolsPath)..\..\..\nanoFramework\v1.0\</NanoFrameworkProjectSystemPath>
+        </PropertyGroup>
+        <Import Project=""$(NanoFrameworkProjectSystemPath)NFProjectSystem.Default.props"" Condition=""Exists('$(NanoFrameworkProjectSystemPath)NFProjectSystem.Default.props')"" />
+        <PropertyGroup>
+        <Configuration Condition="" '$(Configuration)' == '' "">Debug</Configuration>
+        <Platform Condition="" '$(Platform)' == '' "">AnyCPU</Platform>
+        <ProjectTypeGuids>{11A8DD76-328B-46DF-9F39-F559912D0360};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
```

Comment:
`11A8DD76-328B-46DF-9F39-F559912D0360` is it actual new project type?

Reply:
Yes, that's for .NET nanoFramework project type